build.gradle

```diff
@@ -12,7 +11,7 @@ buildscript {
         classpath 'com.vanniktech:gradle-maven-publish-plugin:0.9.0-SNAPSHOT'
         classpath 'com.android.tools.build:gradle:3.5.1'
         classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
-        classpath "org.jetbrains.dokka:dokka-android-gradle-plugin:$dokka_version"
+        classpath "org.jetbrains.dokka:dokka-gradle-plugin:0.10.0"
```

Comment:
I thought having the version number at the top was kinda nice, does it not work with this version?

Reply:
to clarify, I meant: would
```gradle
ext.dokka_version = '0.10.0'
. 
.
.
         classpath "org.jetbrains.dokka:dokka-gradle-plugin:$dokka_version"
```
work?

Reply:
yes, it works, but Android Studio doesn't notify there's an update available
<img width="989" alt="Screen Shot 2020-02-04 at 11 46 24 AM" src="https://user-images.githubusercontent.com/664544/73780801-01193980-4744-11ea-9666-ea934bdff496.png">
<img width="810" alt="Screen Shot 2020-02-04 at 11 46 04 AM" src="https://user-images.githubusercontent.com/664544/73780802-01193980-4744-11ea-98a2-ebd353900f3a.png">

(The underlined kotlin message says the version of the plugin differs from the IDE one)

Reply:
cool, that makes sense. Definitely better to get the notification from Android Studio.